symposion/schedule/models.py

```diff
@@ -51,10 +51,11 @@ class Room(models.Model):
 
     schedule = models.ForeignKey(Schedule, verbose_name=_("Schedule"))
     name = models.CharField(max_length=65, verbose_name=_("Name"))
+    track = models.CharField(max_length=80, verbose_name=_("Track"), default=None, blank=True, null=True)
```

Comment:
I'm not entirely sure I like this approach, but it'll do for now.

If you're up for it, in a later change, I think a more sensible way to do this -- one that would also permit a tutorials track over multiple days.

* Have a `Track` model
* `Track` has a `name`
* `Track` has a `Room`
* `Track` has one or more `Date`s
* There can only be one Track with a given Room and Date.